code/modules/holiday/holidays.dm

```diff
@@ -661,3 +661,102 @@
 /datum/holiday/pride_week/proc/get_floor_tile_color(atom/atom)
 	var/turf/turf = get_turf(atom)
 	return rainbow_colors[(turf.y % rainbow_colors.len) + 1]
+
+/datum/holiday/remembrance_day
+	name = "Remembrance Day"
+	begin_month = NOVEMBER
+	begin_day = 11
+	drone_hat = /obj/item/food/grown/poppy
+
+/datum/holiday/remembrance_day/getStationPrefix()
+	return pick("Peace", "Armistice", "Poppy")
+
+/datum/holiday/victory_day
+	name = "Victory in Europe Day"
+	begin_month = MAY
+	begin_day = 8
```

Comment:
Violates https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#banned-content - this is National Socialist Party of Germany related content.

Reply:
Wasn't even in the PR body, trying to hide this one lol

Reply:
I uh, would have thought the day celebrating victory over the Nazis would have been kosher, but if it's not, I'll remove it

Reply:
> Wasn't even in the PR body, trying to hide this one lol

it's also mentioned in the PR body:
"Remembrance Day and **VE Day** are self explanatory, and fall on November 11th and May 8th, respectively."
VE Day, May 8th, is the celebration of the end of WW2, and was, I thought, self explanatory enough to not require its own callout, same with Remembrance Day.

Reply:
You are assuming a proud American keeps track of bank days not in the greatest nation on Earth